releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -2724,7 +2793,7 @@
   "124933": {
     "commit": "0f584a9b86248bfc62c7d9865d437cb935278bc2",
     "text": "Fixed a bug in the scheduler where it would crash when prefilter returns a non-existent node.",
-    "markdown": "Fixed a bug in the scheduler where it would crash when prefilter returns a non-existent node. ([#124933](https://github.com/kubernetes/kubernetes/pull/124933), [@AxeZhan](https://github.com/AxeZhan)) [SIG Scheduling and Testing]",
+    "markdown": "Fixed a bug in the scheduler where it would crash when prefilter returns a non-existent node. ([#124933](https://github.com/kubernetes/kubernetes/pull/124933), [@AxeZhan](https://github.com/AxeZhan))",
```

Comment:
```suggestion
    "markdown": "Fixed a bug in the scheduler where it would crash when prefilter returns a non-existent node. ([#124933](https://github.com/kubernetes/kubernetes/pull/124933), [@AxeZhan](https://github.com/AxeZhan)) [SIG Scheduling and Testing]",
```